src/common/services/string.js

```diff
@@ -1,6 +1,9 @@
 export const toKebabCase = (str) => {
   return str
-    .match(/[A-Z]{2,}(?=[A-Z][a-z]+[0-9]*|\b)|[A-Z]?[a-z]+[0-9]*|[A-Z]|[0-9]+/g)
+    .match(
+      //  eslint-disable-next-line no-useless-escape
```

Comment:
This should go to global eslint config. A good candidate to update at https://github.com/reactplay/react-play/pull/332

Reply:
Okay, so for now we can keep this and once #332 get's merged we can remove the comment from here.